custom_components/tuya_local/devices/emos_ip300_camera.yaml

```diff
@@ -0,0 +1,442 @@
+name: Camera with light
+products:
+  - id: wauxpj9p977uxhvb
+    manufacturer: EMOS
+    model: IP-300
+entities:
+  - entity: camera
+    dps:
+      - id: 134
+        type: boolean
+        name: motion_enable
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+      - id: 150
+        type: boolean
+        name: record
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+      - id: 115
+        type: base64
+        optional: true
+        persist: false
```

Comment:
I don't think persist: false is desirable here, as it may make the snapshot flash up for a short time only. That is if it is a decodable image, only a few Tuya cameras send actual snapshots (and they are very low resolution), most just give lookup keys for cloud storage.